festipals/controllers/ActController.js

```diff
@@ -0,0 +1,11 @@
+var Act = require('../model/pal');
+
+
+  var createUserAccount = function (req, res) {
```

Comment:
here it's also mixing up pals and acts. create user account probably shouldn't find acts